diff --git a/cmake/CeresConfig.cmake.in b/cmake/CeresConfig.cmake.in
index e5e2976..184d4eb 100644
--- a/cmake/CeresConfig.cmake.in
+++ b/cmake/CeresConfig.cmake.in
@@ -193,17 +193,6 @@ set(CERES_EIGEN_VERSION @EIGEN3_VERSION_STRING@)
 # match and reject with an explanation below.
 find_package(Eigen3 ${CERES_EIGEN_VERSION} QUIET)
 if (EIGEN3_FOUND)
-  if (NOT EIGEN3_VERSION_STRING VERSION_EQUAL CERES_EIGEN_VERSION)
-    # CMake's VERSION check in FIND_PACKAGE() will accept any version >= the
-    # specified version. However, only version = is supported. Improve
-    # usability by explaining why we don't accept non-exact version matching.
-    ceres_report_not_found("Found Eigen dependency, but the version of Eigen "
-      "found (${EIGEN3_VERSION_STRING}) does not exactly match the version of Eigen "
-      "Ceres was compiled with (${CERES_EIGEN_VERSION}). This can cause subtle "
-      "bugs by triggering violations of the One Definition Rule. See the "
-      "Wikipedia article http://en.wikipedia.org/wiki/One_Definition_Rule "
-      "for more details")
-  endif ()
   ceres_message(STATUS "Found required Ceres dependency: "
     "Eigen version ${CERES_EIGEN_VERSION} in ${EIGEN3_INCLUDE_DIRS}")
 else (EIGEN3_FOUND)
